rclcpp/minimal_client/not_composable.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include "rclcpp/rclcpp.hpp"
+#include "examples_rclcpp_minimal_service/srv/add_two_ints.hpp"
+
+using Service = examples_rclcpp_minimal_service::srv::AddTwoInts;
+using Request = Service::Request;
```

Comment:
Same comment as in the other file, I get what you're trying to do here with shorter lines and simpler names, but I think `AddTwoInts::Request` is so much more informative than `Request` or `Service::Request`, which require me to look up the translation.